github4s/src/main/scala/github4s/instances.scala

```diff
@@ -16,17 +16,19 @@
 
 package github4s
 
-import cats.instances.FutureInstances
 import cats.{Eval, FlatMap, Id, Monad, MonadError}
+import cats.instances.FutureInstances
 import github4s.free.interpreters._
 import scala.concurrent.{ExecutionContext, Future}
 
-object implicits
+trait Instances
     extends FutureCaptureInstance
     with EvalInstances
     with IdInstances
     with FutureInstances
 
+object implicits1 extends Instances
```

Comment:
why you choose `implicits1` @BenFradet ?

Reply:
I wanted to avoid the clash with `implicits` since these are internal I went for `implicits`, if you have an idea for a better name I'm :+1:

Reply:
A friend of us always said that the two hardest things in the engineering are cache invalidation and naming 😂